File: Open3D/01_fundamentos/python/main.py
import open3d as o3d
import numpy as np
pts=np.random.default_rng(7).normal(size=(5000,3)); pc=o3d.geometry.PointCloud(o3d.utility.Vector3dVector(pts)); print(pc); o3d.io.write_point_cloud("output.ply",pc)

File: Open3D/01_fundamentos/cpp/main.cpp
#include <open3d/Open3D.h>
#include <random>
int main(){ auto pc=std::make_shared<open3d::geometry::PointCloud>(); std::mt19937 g(7); std::normal_distribution<double>d; for(int i=0;i<5000;i++) pc->points_.emplace_back(d(g),d(g),d(g)); open3d::io::WritePointCloud("output.ply",*pc); }
